src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -1076,7 +1077,11 @@ bool HMICapabilitiesImpl::load_capabilities_from_file() {
           Formatters::CFormatterJsonBase::jsonValueToObj(
               navigation_capability, navigation_capability_so);
           set_navigation_capability(navigation_capability_so);
+          if (!navigation_capability_so.empty()) {
+            set_navigation_supported(true);
+          }
         }
+        set_phone_call_supported(false);
```

Comment:
I think setting these to false  before checking for the capability is unnecessary since those values are already initialized to false by the constructor.

Reply:
@shoamano83

Reply:
Thanks for your comment.
This call is added to reset the flag when HMICapabilitiesImpl::load_capabilities_from_file() is called more than once. I understand that such situation won’t happen in production, but it does in unit testing. It is called during HMICapabilitiesTest() constructor, then during hmi_capabilities_test->LoadCapabilitiesFromFile() call in each test.

I’ll try if I can remove this double-loading issue in the unit tests.

Reply:
@JackLivio Fixed with https://github.com/smartdevicelink/sdl_core/pull/1763/commits/fc201151a0fda0eaef0b8cf300afc4a96239255b. Please kindly review again.